components/org.wso2.carbon.identity.application.authenticator.passive.sts/src/main/java/org/wso2/carbon/identity/application/authenticator/passive/sts/PassiveSTSAuthenticator.java

```diff
@@ -146,6 +149,16 @@ public String getContextIdentifier(HttpServletRequest request) {
                     log.error("Exception while URL decoding the Relay State", e);
                 }
             }
+
+            identifier = request.getParameter(PassiveSTSConstants.HTTP_PARAM_PASSIVE_STS_LOGOUT);
+
+            if (identifier != null) {
+                try {
```

Comment:
add debug logs to here and earlier if block to indicate where it found the identifier.

Reply:
Its ok to print identifier itself